src/server.php

```diff
@@ -30,14 +31,41 @@
 //-----------------------------------------------------------------------------
 class openFormat extends webServiceServer {
 
+  private $redis_cache;
+
+  /**
+   * constructor
+   */
   public function __construct() {
     webServiceServer::__construct('openformat.ini');
+    $cachesettings = $this->config->get_section('rediscache');
+    $this->redis_cache = new RedisClusterCache($cachesettings['cache_host']);
+  }
+
+  /**
+   * Make a cache key from pids and given display
+   *
+   * @param array $pids
+   * @param $display
+   *
+   * @return string
+   */
+  private function make_cache_key(array $pids, $display) {
+    $cache_key = "";
```

Comment:
Det er en overvejelse at sætte et præfix på cache_key, fx OF, så en ander service som bruger samme cluster, ikke ved et uheld laver samme nøgle. Det er nok ikke særligt sandsynligt, men måske mere en no-brainer sikkerhed.

Reply:
jo - jeg laver et prefix